nltk/downloader.py

```diff
@@ -2258,7 +2258,8 @@ def update():
     parser.add_option("-e", "--exit-on-error", dest="halt_on_error", action="store_true",
         default=False, help="exit if an error occurs")
     parser.add_option("-u", "--url", dest="server_index_url",
-        default=None, help="download server index url")
+        default=os.environ.get('NLTK_DOWNLOAD_URL', None),
```

Comment:
@zxiiro I mean that you are using `os.environ.get('NLTK_DOWNLOAD_URL', None)`, but I think that `None` is not needed when using `get()`, since it's already the default value.